exporter/dorisexporter/README.md

```diff
@@ -0,0 +1,69 @@
+# Apache Doris Exporter
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: traces, metrics, logs   |
+| Distributions | [contrib] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Aexporter%2Fdoris%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Aexporter%2Fdoris) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Aexporter%2Fdoris%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Aexporter%2Fdoris) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@atoulme](https://www.github.com/atoulme) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+This exporter supports sending traces, metrics, and logs data to [Apache Doris](https://doris.apache.org/).
+
+## Configuration
+
+The following configuration options are supported:
+
+* `endpoint`
+  * `http` The http stream load address.
+  * `tcp` The mysql protocol tcp address; ignored if create_schema is false.
```

Comment:
this is confusing, as we tend to use endpoint directly most of the time. Can we do this instead:
```
* `endpoint` the Apache Doris stream load endpoint, as an HTTP URL.
* `database_endpoint` the MySQL database endpoint to create the schema ; ignored if create_schema is false.

Reply:
on that note, if the schema creation is a one shot or manual step, it might deserve to be separated into its own mechanism. Will this manage upgrades?